core-bundle/src/Resources/contao/drivers/DC_Table.php

```
@@ -982,6 +991,8 @@ protected function copyChilds($table, $insertID, $id, $parentId)
 												->execute($id);
 				}
 
+				$this->denyAccessIfDisallowed(self::PERMISSION_COPY, new DataContainerSubject($v, $objCTable->id));
```

shouldn't this be in the while-loop in the line below?

---

I guess so, and it should also ignore the exception. See 7042bcbf4f5ed3096c9be157e1551a8f5e3de0ea.